[0m21:49:41.220371 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E0EF7F500>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E0F359EE0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E0BBC7B30>]}


============================== 21:49:41.226372 | 4d7553a2-6e2a-4ce6-9791-5fe13f155888 ==============================
[0m21:49:41.226372 [info ] [MainThread]: Running with dbt=1.8.9
[0m21:49:41.227372 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ubicase01\\.dbt', 'fail_fast': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'debug': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'static_parser': 'True', 'log_format': 'default', 'introspect': 'True', 'target_path': 'None', 'invocation_command': 'dbt init', 'send_anonymous_usage_stats': 'True'}
[0m21:50:47.953649 [debug] [MainThread]: Starter project path: C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\.venv\Lib\site-packages\dbt\include\starter_project
[0m21:50:48.058433 [info ] [MainThread]: 
Your new dbt project "datawarehouse" was created!

For more information on how to configure the profiles.yml file,
please consult the dbt documentation here:

  https://docs.getdbt.com/docs/configure-your-profile

One more thing:

Need help? Don't hesitate to reach out to us via GitHub issues or on Slack:

  https://community.getdbt.com/

Happy modeling!

[0m21:50:48.060438 [info ] [MainThread]: Setting up your profile.
[0m21:52:15.078437 [info ] [MainThread]: Profile datawarehouse written to C:\Users\ubicase01\.dbt\profiles.yml using target's profile_template.yml and your supplied values. Run 'dbt debug' to validate the connection.
[0m21:52:15.084845 [debug] [MainThread]: Command `dbt init` succeeded at 21:52:15.084845 after 153.97 seconds
[0m21:52:15.085846 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E0F6E1F70>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E098E1CA0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000016E0F618D70>]}
[0m21:52:15.086848 [debug] [MainThread]: Flushing usage events
[0m00:28:01.682709 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC27BFE90>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC27BF500>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC2FAEE10>]}


============================== 00:28:01.682709 | 4e0ee11d-e44f-472f-bbff-d2304cec3426 ==============================
[0m00:28:01.682709 [info ] [MainThread]: Running with dbt=1.8.9
[0m00:28:01.698335 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ubicase01\\.dbt', 'fail_fast': 'False', 'debug': 'False', 'log_path': 'logs', 'version_check': 'True', 'warn_error': 'None', 'use_colors': 'True', 'use_experimental_parser': 'False', 'empty': 'False', 'quiet': 'False', 'no_print': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'invocation_command': 'dbt run', 'introspect': 'True', 'static_parser': 'True', 'target_path': 'None', 'log_format': 'default', 'send_anonymous_usage_stats': 'True'}
[0m00:28:01.698335 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m00:28:01.698335 [debug] [MainThread]: Command `dbt run` failed at 00:28:01.698335 after 0.12 seconds
[0m00:28:01.698335 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC2D47E60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC2F298E0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000011AC25C0EF0>]}
[0m00:28:01.698335 [debug] [MainThread]: Flushing usage events
[0m11:14:33.593343 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029140C0F620>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002913DF66960>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029140826510>]}


============================== 11:14:33.593343 | 36154a38-ac59-493a-8704-0ac9852db1d6 ==============================
[0m11:14:33.593343 [info ] [MainThread]: Running with dbt=1.8.9
[0m11:14:33.609040 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'write_json': 'True', 'log_cache_events': 'False', 'partial_parse': 'True', 'cache_selected_only': 'False', 'profiles_dir': 'C:\\Users\\ubicase01\\.dbt', 'debug': 'False', 'version_check': 'True', 'log_path': 'logs', 'warn_error': 'None', 'fail_fast': 'False', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'None', 'log_format': 'default', 'invocation_command': 'dbt debug', 'static_parser': 'True', 'introspect': 'True', 'target_path': 'None', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'send_anonymous_usage_stats': 'True'}
[0m11:14:33.634925 [info ] [MainThread]: dbt version: 1.8.9
[0m11:14:33.635925 [info ] [MainThread]: python version: 3.12.3
[0m11:14:33.637928 [info ] [MainThread]: python path: C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\.venv\Scripts\python.exe
[0m11:14:33.638926 [info ] [MainThread]: os info: Windows-10-10.0.19044-SP0
[0m11:14:33.967867 [info ] [MainThread]: Using profiles dir at C:\Users\ubicase01\.dbt
[0m11:14:33.968866 [info ] [MainThread]: Using profiles.yml file at C:\Users\ubicase01\.dbt\profiles.yml
[0m11:14:33.969980 [info ] [MainThread]: Using dbt_project.yml file at C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\dbt_project.yml
[0m11:14:33.970980 [info ] [MainThread]: Configuration:
[0m11:14:33.971782 [info ] [MainThread]:   profiles.yml file [[31mERROR invalid[0m]
[0m11:14:33.972795 [info ] [MainThread]:   dbt_project.yml file [[31mERROR not found[0m]
[0m11:14:33.973794 [info ] [MainThread]: Required dependencies:
[0m11:14:33.974796 [debug] [MainThread]: Executing "git --help"
[0m11:14:33.999797 [debug] [MainThread]: STDOUT: "b"usage: git [-v | --version] [-h | --help] [-C <path>] [-c <name>=<value>]\n           [--exec-path[=<path>]] [--html-path] [--man-path] [--info-path]\n           [-p | --paginate | -P | --no-pager] [--no-replace-objects] [--bare]\n           [--git-dir=<path>] [--work-tree=<path>] [--namespace=<name>]\n           [--super-prefix=<path>] [--config-env=<name>=<envvar>]\n           <command> [<args>]\n\nThese are common Git commands used in various situations:\n\nstart a working area (see also: git help tutorial)\n   clone     Clone a repository into a new directory\n   init      Create an empty Git repository or reinitialize an existing one\n\nwork on the current change (see also: git help everyday)\n   add       Add file contents to the index\n   mv        Move or rename a file, a directory, or a symlink\n   restore   Restore working tree files\n   rm        Remove files from the working tree and from the index\n\nexamine the history and state (see also: git help revisions)\n   bisect    Use binary search to find the commit that introduced a bug\n   diff      Show changes between commits, commit and working tree, etc\n   grep      Print lines matching a pattern\n   log       Show commit logs\n   show      Show various types of objects\n   status    Show the working tree status\n\ngrow, mark and tweak your common history\n   branch    List, create, or delete branches\n   commit    Record changes to the repository\n   merge     Join two or more development histories together\n   rebase    Reapply commits on top of another base tip\n   reset     Reset current HEAD to the specified state\n   switch    Switch branches\n   tag       Create, list, delete or verify a tag object signed with GPG\n\ncollaborate (see also: git help workflows)\n   fetch     Download objects and refs from another repository\n   pull      Fetch from and integrate with another repository or a local branch\n   push      Update remote refs along with associated objects\n\n'git help -a' and 'git help -g' list available subcommands and some\nconcept guides. See 'git help <command>' or 'git help <concept>'\nto read about a specific subcommand or concept.\nSee 'git help git' for an overview of the system.\n""
[0m11:14:34.000796 [debug] [MainThread]: STDERR: "b''"
[0m11:14:34.001794 [info ] [MainThread]:  - git [[32mOK found[0m]

[0m11:14:34.002795 [info ] [MainThread]: Connection test skipped since no profile was found
[0m11:14:34.003795 [info ] [MainThread]: [31m2 checks failed:[0m
[0m11:14:34.003795 [info ] [MainThread]: Could not load dbt_project.yml
Profile loading failed for the following reason:
Runtime Error
  Credentials in profile "caseabcbank", target "dev" invalid: 'schema' is a required property

Runtime Error
  Credentials in profile "datawarehouse", target "dev" invalid: 'schema' is a required property


[0m11:14:34.005796 [info ] [MainThread]: Project loading failed for the following reason:
 project path <C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\dbt_project.yml> not found

[0m11:14:34.007795 [debug] [MainThread]: Command `dbt debug` failed at 11:14:34.006794 after 0.52 seconds
[0m11:14:34.007795 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x000002913D037B60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029140D54B60>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000029140AF6660>]}
[0m11:14:34.008798 [debug] [MainThread]: Flushing usage events
[0m08:43:15.355788 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'start', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A117EF7A0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A0F2DC3B0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A0F2DC980>]}


============================== 08:43:15.361788 | a32d02a6-2500-4e68-bac3-d6ff7c770142 ==============================
[0m08:43:15.361788 [info ] [MainThread]: Running with dbt=1.8.9
[0m08:43:15.363790 [debug] [MainThread]: running dbt with arguments {'printer_width': '80', 'indirect_selection': 'eager', 'log_cache_events': 'False', 'write_json': 'True', 'partial_parse': 'True', 'cache_selected_only': 'False', 'warn_error': 'None', 'fail_fast': 'False', 'profiles_dir': 'C:\\Users\\ubicase01\\.dbt', 'log_path': 'logs', 'debug': 'False', 'version_check': 'True', 'use_colors': 'True', 'use_experimental_parser': 'False', 'no_print': 'None', 'quiet': 'False', 'empty': 'False', 'log_format': 'default', 'static_parser': 'True', 'introspect': 'True', 'warn_error_options': 'WarnErrorOptions(include=[], exclude=[])', 'target_path': 'None', 'invocation_command': 'dbt run', 'send_anonymous_usage_stats': 'True'}
[0m08:43:15.364790 [error] [MainThread]: Encountered an error:
Runtime Error
  No dbt_project.yml found at expected path C:\Users\ubicase01\Desktop\Entrega\Case-ABC-Bank\dbt_project.yml
  Verify that each entry within packages.yml (and their transitive dependencies) contains a file named dbt_project.yml
  
[0m08:43:15.365791 [debug] [MainThread]: Command `dbt run` failed at 08:43:15.365791 after 0.16 seconds
[0m08:43:15.366792 [debug] [MainThread]: Sending event: {'category': 'dbt', 'action': 'invocation', 'label': 'end', 'context': [<snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A110DDAC0>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A1196C170>, <snowplow_tracker.self_describing_json.SelfDescribingJson object at 0x0000019A1196CE60>]}
[0m08:43:15.367789 [debug] [MainThread]: Flushing usage events
